onmt/Optim.py

```diff
@@ -31,13 +31,9 @@ class Optim(object):
     # https://arxiv.org/pdf/1706.03762.pdf, particularly the value beta2=0.98
     # was used there however, beta2=0.999 is still arguably the more
     # established value, so we use that here as well
-    def __init__(self, method, lr, max_grad_norm,
-                 lr_decay=1, start_decay_at=None,
-                 beta1=0.9, beta2=0.999,
-                 adagrad_accum=0.0,
-                 decay_method=None,
-                 warmup_steps=4000,
-                 model_size=None):
+    def __init__(self, method, lr, max_grad_norm, lr_decay=0.9, start_decay_at=None,
+                 beta1=0.9, beta2=0.999, adagrad_accum=0.0,
+                 decay_method=None, warmup_steps=4000, model_size=None):
```

Comment:
is there a reason to change its default setting?

Reply:
I was experimenting to see if this caused the drop in performance, and now we know it wasn't, so I've changed it back to the original settings.